src/components/OrgView/OrgView.js

```diff
@@ -99,6 +100,14 @@ function OrgView({
             justify-content: space-between;
             background: ${theme.surface};
             box-shadow: 0 2px 3px rgba(0, 0, 0, 0.05);
+
+            ${menuPanelOpen && iOS
+              ? `
+                position: absolute;
```

Comment:
This is already styled to be `position: relative`, so we do need the `position: absolute`?

Reply:
Yes, bringing in the conversation from aragon.chat:

>The changes specific for iOS
>are very weird
>there is a nested div->div->div tree structure
>the 3rd div is positioned absolute and in Chrome and Firefox and Safari gets rendered on >top on anything up the tree till a position relative is found
>but, in iOS it gets rendered below the sibling of the element with position relative
>and it did not work to put a `z-index` on that sibling, I had to make it `absolute` and >simulate its height

Reply:
Perhaps we can summarize this and add it as a comment :)